app/views/posts/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1>A User Specific Post</h1>
\ No newline at end of file
+<h1>A User Specific Post</h1>
+<div class="posts">
+  <div class="card-info">
+    <h3> <%=@post.title%> by <%= @user.name%> </h3>
```

Comment:
- [ ] You have introduced a bug here now, this leads to an error page, kindly fix it 
![image](https://github.com/Johnadibe/blog_app/assets/62705947/7531cc95-5eb8-4997-a3e4-847df68cdcc9)